action.yml

```diff
@@ -246,6 +246,7 @@ runs:
         WP_CONFIG_PATH="${WP_TESTS_DIR}/wp-tests-config.php"
 
         echo "define( 'IS_UNIT_TEST', true );" >> "${WP_CONFIG_PATH}"
+        echo "define( 'VIP_GO_APP_ENVIRONMENT', 'staging' );" >> "${WP_CONFIG_PATH}"
```

Comment:
Why is this needed? I don't see any check of the              `VIP_USE_NEXT_OBJECT_CACHE_DROPIN` constant in the new loader, just a check for the PHP extension

Reply:
Thanks for pointing it out, Seems like I had mistaken it as a requirement. I tried by removing the constant and pipeline seems fine